crates/web-sys/webidls/unstable/AudioContext.webidl

```diff
@@ -0,0 +1,6 @@
+partial interface AudioContext {
+  readonly        attribute DOMString sinkId;
+
+  [Throws]
```

Comment:
Where did you get that this throws?
The spec doesn't mention it: https://webaudio.github.io/web-audio-api/#dom-audiocontext-setsinkid-domstring-or-audiosinkoptions-sinkid.
Chromium doesn't do it either: https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/web_tests/external/wpt/interfaces/webaudio.idl;l=83?q=setsinkid&ss=chromium%2Fchromium%2Fsrc.

Reply:
Ah, yeah I've not properly understood the mechanism then, I was under the impression promises have the possibility of throwing and I needed it here like I saw else - will remove 👍

Reply:
Just to clarify, `Promise`s throwing or not is handled by the `Promise` type itself, no need to label it in the WebIDL at all.

Reply:
FWIW this was what I was looking at: [MDN setSinkId Exceptions](https://developer.mozilla.org/en-US/docs/Web/API/AudioContext/setSinkId#exceptions)

Reply:
> Promises throwing or not is handled by the Promise type itself, no need to label it in the WebIDL at all.
That's helpful, thanks!

Reply:
> FWIW this was what I was looking at: [MDN setSinkId Exceptions](https://developer.mozilla.org/en-US/docs/Web/API/AudioContext/setSinkId#exceptions)

I just tried it myself, I can confirm that the exceptions listed here are rejected promises and aren't thrown on the function itself.